src/index.html

```diff
@@ -10,23 +10,33 @@
     <div class="login-wrap">
       <form>
         <div class="field">
-          <input type="text" name="firstName"
-                 class="field-text" autocomplete="off"
-                 id="sign-up-name" autofocus>
+          <input type="text"
+                 name="firstName"
+                 class="field-text"
+                 autocomplete="off"
+                 id="sign-up-name" 
+                 autofocus
+                 placeholder="">
         </div>
         <div class="field">
-          <input type="text" name="lastName"
-                 class="field-text" id="sign-up-last-name"
+          <input type="text"
+                 name="lastName"
+                 class="field-text"
+                 id="sign-up-last-name"
                  autocomplete="off">
         </div>
         <div class="field">
-          <input type="text" name="email"
-                 class="field-text" id="sign-up-email"
+          <input type="text"
+                 name="email"
+                 class="field-text"
+                 id="sign-up-email"
                  autocomplete="off">
```

Comment:
The same problem as previously. If we have 3 or more attributes, they must be on a new line, see the example. This will make your code more readable. Check it out everywhere
```suggestion
          <input
            type="text"
            name="email"
            class="field-text"
            id="sign-up-email"
            autocomplete="off"
          >
```